executor/src/main/java/org/k8loud/executor/action/openstack/HorizontalScalingAction.java

```diff
@@ -0,0 +1,61 @@
+package org.k8loud.executor.action.openstack;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.inject.Module;
+import data.ExecutionExitCode;
+import data.ExecutionRS;
+import org.jclouds.ContextBuilder;
+import org.jclouds.logging.slf4j.config.SLF4JLoggingModule;
+import org.jclouds.openstack.nova.v2_0.NovaApi;
+import org.jclouds.openstack.nova.v2_0.domain.Flavor;
+import org.jclouds.openstack.nova.v2_0.domain.Server;
+import org.jclouds.openstack.nova.v2_0.features.FlavorApi;
+import org.jclouds.openstack.nova.v2_0.features.ServerApi;
+import org.jclouds.openstack.v2_0.domain.Resource;
+import org.k8loud.executor.action.Action;
+
+import java.util.Map;
+
+public class HorizontalScalingAction extends Action {
+    private static final String REGION = "region";
+    public static final String SERVER_ID = "id";
+    private final NovaApi novaApi;
+
+    public HorizontalScalingAction(Map<String, String> params) {
```

Comment:
params are not used.
I believe that `REGION, SERVER_ID, endpoint, credentials` should be passed as params.
It's okay for now as the feature is yet to be tested but we should keep it in mind.

Reply:
Yes I know, but as I said. It's just a schema. I need to be sure that it's generally working